config/environments/profiling.rb

```diff
@@ -3,7 +3,7 @@
 Rails.application.configure do
   config.public_file_server.enabled = true
 
-  logger           = ActiveSupport::Logger.new(STDOUT)
+  logger           = ActiveSupport::Logger.new('log/profiling.log')
```

Comment:
afaik we moved to stdout because we configured Docker (or Docker Compose?) to capture stdout and log it all to a file on the host, which has log rotation set up. Is that still the case?

Reply:
Yes, but when you run derailed stuff the output from rails basically removes all the information that it gives you.  I'm pretty sure you can have rails do log rotation as well if you think it would be beneficial.

Reply:
Did we determine that it's required? Otherwise it will just keep increasing in size forever until it crashes the box right?

Reply:
Well this only changes the profiling environment, which should never be ran long term on a production box, if ever.

Reply:
Ah, I missed the filename and thought this was for production! Nevermind then.